src/Builder/Type/CombinedFieldQuery.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MongoDB\Builder\Type;
+
+use InvalidArgumentException;
+use MongoDB\BSON\Document;
+use MongoDB\BSON\Serializable;
+use stdClass;
+
+use function get_debug_type;
+use function is_array;
+use function sprintf;
+
+/**
+ * List of filters that apply to the same field path.
+ */
+class CombinedFieldQuery implements FieldQueryInterface
+{
+    /** @param list<FieldQueryInterface|Serializable|array|stdClass> $filters */
+    public function __construct(public array $filters)
```

Comment:
Is this intended to be a public property? Should it be `readonly` if so?

Reply:
Yes. Fixed.